File: client/src/app/_models/libraryLedger.ts
import { AppUser } from "./appUser";
import { Book } from "./book";

export interface LibraryLedger {
    id: number;
    user: AppUser;
    book: Book;
    created: string;
    lentFrom: string;
    lentTo: string;
    lentOut: boolean;
    overdue: boolean;
}

File: API/Entities/LibraryLedger.cs
namespace API.Entities
{
    public class LibraryLedger
    {
        public int Id { get; set; }
        public AppUser User { get; set; }
        public Book Book { get; set; }
        public DateTime Created { get; set; }  = DateTime.Now;
        public DateTime LentFrom { get; set; }
        public DateTime LentTo { get; set; }
        public bool LentOut { get; set; }
        public bool Overdue { get; set; }

    }
}